src/social_db/data/delete/mod.rs

```diff
@@ -0,0 +1,115 @@
+use std::str::FromStr;
+
+use color_eyre::eyre::{ContextCompat, WrapErr};
+use near_cli_rs::common::{CallResultExt, JsonRpcClientExt};
+
+mod sign_as;
+
+#[derive(Debug, Clone, interactive_clap::InteractiveClap)]
+#[interactive_clap(input_context = crate::GlobalContext)]
+#[interactive_clap(output_context = DeleteContext)]
+pub struct Delete {
+    /// For which key do you want to delete information?
+    key: String,
+    #[interactive_clap(named_arg)]
+    /// Specify signer account ID
+    sign_as: self::sign_as::Signer,
+}
+
+#[derive(Clone)]
+pub struct DeleteContext(self::sign_as::PreparedSignerContext);
+
+impl DeleteContext {
+    pub fn from_previous_context(
+        previous_context: crate::GlobalContext,
+        scope: &<Delete as interactive_clap::ToInteractiveClapContextScope>::InteractiveClapContextScope,
+    ) -> color_eyre::eyre::Result<Self> {
+        let account_id = near_cli_rs::types::account_id::AccountId::from_str(
+            scope.key.split('/').map(|s| s.trim()).collect::<Vec<_>>()[0],
```

Comment:
Use `.split_once()` instead of `.split()[0]`

Reply:
Done